### PR TITLE
Fix: Issue related to autocompletion using company caused by pcomplete

### LINKED ISSUE #12024: Ledger mode account completion broken when auto-complete is on
#### Description :octocat:
In ledger mode, cycling through accounts using the tab key when auto-complete is on is broken, by the addition of a space after each tab press. Sub accounts should not be separated by a space between the end of the word and the colon; if a space is added, completion of the subaccount is stopped. This is different from the behaviour of ledger mode in regular Emacs (or when auto-complete is off), where pressing the tab does not add a space, and thus if a subaccount is present, completion can continue. 

#### Reproduction guide :beetle:
- Start Emacs
- Turn auto-completion on
- Add new transaction (SPC m a)
- Start typing a subaccount
- Press tab to continue

*Observed behaviour:* :eyes: :broken_heart:
Account name is followed by a space

*Expected behaviour:* :heart: :smile:
Account name should not be followed by a space, and completion option for any subaccount should be available

#### System Info :computer:
- OS: darwin
- Emacs: 26.1
- Spacemacs: 0.200.13
- Spacemacs branch: master (rev. c7a103a77)
- Graphic display: nil
- Distribution: spacemacs
- Editing style: hybrid
- Completion: helm
- Layers:
```elisp
(auto-completion better-defaults bibtex emacs-lisp finance git helm latex markdown org python semantic
                 (shell :variables shell-default-height 30 shell-default-position 'bottom)
                 (spell-checking :variables enable-flyspell-auto-completion t)
                 syntax-checking version-control)
```
- System configuration features: NOTIFY ACL GNUTLS LIBXML2 ZLIB THREADS

#### Backtrace :paw_prints:
```
<<BACKTRACE IF RELEVANT>>
```

### PR DESCRIPTION
**Pull request by larebsyed**
This PR fixes the issue reported in here:

Fixes #12024

### CODE REVIEW

File: layers/+tools/finance/packages.el [hunk 1]
```diff
@@ -60,4 +60,5 @@
       ;; more info: https://github.com/emacs-evil/evil/issues/301
       ;; TODO remove this hack if the limitation is removed upstream
       (add-hook 'ledger-mode-hook 'evil-normalize-keymaps)
+      (setq pcomplete-termination-string "")
```
Comment (Alexander-Miller) [inline layers/+tools/finance/packages.el]: You are changing the global value here. It should instead be changed locally in a hook.

Comment (larebsyed) [reply layers/+tools/finance/packages.el]: @Alexander-Miller Thanks for review. Ill incorporate the changes pointed out.

Comment (larebsyed) [reply layers/+tools/finance/packages.el]: @Alexander-Miller  Done

**Comment by Alexander-Miller** [comment]
Spacemacs' development has gone quiet for the last month or two. Either it's in hibernation for some reason, or the maintainers are busy preparing the next stable release. Either way you should expect this PR to stay quiet for a while.

**Comment by duianto** [comment]
@larebsyed Could you add an entry to: https://github.com/syl20bnr/spacemacs/blob/develop/CHANGELOG.develop#finance

**Comment by larebsyed** [comment]
@duianto Btw why did we cherry pick instead of merge? Any particular reason behind it. Asking just for knowledge.

**Comment by duianto** [comment]
bmag, said:
>it's for keeping the history linear. since most PRs are small, it'd be annoying to have many small "diverge-and-merge"s in the history graph

**Pull request closed without merging**